magento/adobe-stock-integration
# Locate functionality fix and ACL renaming

### Description (*)

Waiting for data update before selecting an image in the grid during "Open in Media Gallery" functionality

### Fixed Issues (if relevant)
Fixes https://github.com/magento/adobe-stock-integration/issues/1791
Also corrected the names for WebAPI ACL

### Manual testing scenarios (*)

Play with Open in Media Gallery, Save Preview, Save licensed functionality having different grid states and saving/opening images from different folders.
The image should be correctly located and selected
